### ISSUE TITLE
Bug: Gx:Angle in GxTrack causes TypeError

### ISSUE THREAD

**Issue by uzzell** [eisoldt/simplekml#11]
**Reproducible Example**
```
import simplekml
trk = simplekml.GxTrack()
trk.newgxangle(0)
trk.__str__()
```

**Issue Description**
The snippet above produces a TypeError:
```
File "C:\Users\uzzell\AppData\Local\Programs\Python\Python39\lib\site-packages\simplekml\featgeom.py", line 2345, in __str__
    angle_str = ' '.join(map(str, angle))
TypeError: 'int' object is not iterable
```

If one adds a `gx:angle` to a `GxTrack`, the track's `__str__()` function no longer works.  This makes it impossible to save a KML file that contains angles.

**Expected Behavior**
No error message.  I would expect the result of `trk.__str__()` to be something like `"<gx:Track><gx:angles>0</gx:angles></gx:Track>"`.

**Additional information**
The relevant portion of `feat_geom.py` has
```
        for angle in self.gxangles:
            angle_str = ' '.join(map(str, angle))
            buf.append("<gx:angles>{0}</gx:angles>".format(angle_str))
```
Removing `angle_str = ' '.join(map(str, angle))` and replacing `format(angle_str)` with `format(angle)` looks like it would solve the problem.

**Comment by uzzell** [comment]
Closed by #12 .

**Comment by uzzell** [comment]
Closed by #12 .
